src/Symfony/Component/Console/Tester/CommandTester.php

```diff
@@ -129,4 +135,29 @@ public function getStatusCode()
     {
         return $this->statusCode;
     }
+
+    /**
+     * Sets the user inputs.
+     *
+     * @param array An array of strings representing each input
+     *              passed to the command input stream.
+     *
+     * @return CommandTester
+     */
+    public function setInputs(array $inputs)
+    {
+        $this->inputs = $inputs;
+
+        return $this;
+    }
+
+    private static function createStream(array $inputs)
```

Comment:
@fabpot : Did you mean `static` ?
I guess it makes sense, as this method looks like a utility one.